### DISCUSSION TITLE
Unreachable "unresolved" conversations block PR merge

### DISCUSSION THREAD

**Discussion by relsqui** [community/community#10592] [Pull Requests]
I reviewed my teammate's PR and left some questions. They later did a local rebase and force pushed, which left me unable to reach the review conversations, and we can't merge the PR because the conversations are unresolved. They eventually found them again after digging through the hundreds of deploy notifications on the PR page (we run a big plan/test matrix), but that was a terrible user experience.

P.S. And it still sucks to have to participate in a public forum to give one-off UX feedback.

**Comment by jgavris** [comment]
+1 on this ... clicking on an unresolved conversation from the tab after a rebase / force push is a 'not found' You have to expand the 'hidden items' accordion in the PR to get to it.

**Reply by stevenbrix** [reply]
where is this accordian? i can't find it

**Reply by laser** [reply]
Perhaps unsurprisingly, the "hidden items" control appears to be hidden for me as well. Where is this darned thing?

**Reply by laser** [reply]
Ah, I found it. See screenshot. It's in the main PR-view, specifically the list of comments that folks have left on your PR.

![Screenshot 2023-10-02 at 2 43 21 PM](https://github.com/community/community/assets/884507/5bfc586d-fb77-44fa-9792-4afa316e6402)

**Reply by LazarQt** [reply]
Not very user friendly to hide an unresolved comment like this. Would appreciate some changes.

**Comment by x-stefan** [comment]
If you run into this issue (like me): Close the PR and create a new one.

Seems like the only way to resolve this issue quickly.

**Comment by KatrinaHoffert** [comment]
In one case where I encountered this, I managed to find it attached to the first commit of the several commits in the PR. It did not show up in the default view showing changes from all commits. It did show up on a middle commit that happened to change that file, too. It was a file level comment. My best guess is that it's because the file was changed multiple times but not in the final commit? Either way, it was super obscure to find.

**Comment by bborad** [comment]
just go to commit Commits tab and check all commits one by one and resolved any unresolved conversation
